docs/faq.rst

```diff
@@ -123,3 +123,25 @@ Yes, these techniques are complementary.
 Multiband fMRI leads to collecting multiple slices within a volume  simultaneously, while multi-echo
 fMRI is instead related to collecting multiple unique volumes.
 These techniques can be combined to reduce the TR in a multi-echo sequence.
+
+********************************************************************************
+[ME-fMRI] How do field maps and distortion correction work with multi-echo fMRI?
+********************************************************************************
+
+There are many different approaches to susceptibility distortion correction out there- far too many to discuss here.
+However, the good news is that distortion does not typically change across echoes in multi-echo fMRI.
+In most cases, the ``EffectiveEchoSpacing`` is the same across echoes,
+so distortion will remain relatively constant, even as dropout increases with echo time.
+
+What this means is that, in the vast majority of multi-echo datasets,
+standard distortion correction methods will work, and should be applied in the same manner on all echoes.
+For example, if you acquire a blip-up/blip-down set of images for all of your echo times,
+you should use the first echo time's images to generate the undistortion transform,
+as long as that first echo has sufficient gray/white constrast to be useful for alignment
+(in which case, use the earliest echo that does have good contrast).
+
+One edge case that will become more common over time is ultra-high resolution multi-echo fMRI.
+In ultra-high resolution data (voxels <1 mm3), minute differences in distortion can become meaningful.
```

Comment:
Honestly I'm not even sure it bears mentioning. I think even with ultra-high res fMRI the B0 distortions would be the same, there may just be other effects kind of look like distortion that vary from echo to echo. But maybe I am wrong. I think <1mm is fine. It'll be a while before folks are regularly hitting that without a lot of effort (for multiecho).

Reply:
Oh, okay. I think I misunderstood what you were saying in the thread. Do you know what these other effects might be?

Reply:
I'll probably make the text a little vaguer anyway.

Reply:
https://www.biorxiv.org/content/10.1101/2021.11.25.470023v3.full#F10 is an example of it - basically, over echoes, there is a blood flow artifact that moves around (as in the blood signal shifts over TE), but usually we wouldn't see this i typical fMRI for various reasons. I just didn't want to go on the record saying that there was no effect when I knew there was at least a possibility of a small effect. I've looked for it in a few different datasets and never seen it, but usually blood isn't super visible anyway, so all these words just to say, mostly that

I don't think people need to worry about it. Maybe even ever. But as my MRI knowledge is incomplete, I thought it worth hedging a bit, in case someone digs up some effect sometime. With that comment in there, at least people could in theory be on the look out for edge cases, which is good.

Reply:
Ah, that's good to know. I've removed the mentions of ultra-high resolution data for now. We can add more detailed info in the future.